src/rust-2024/rustfmt-overflow-delimited-expr.md

```diff
@@ -1,10 +1,90 @@
 # Rustfmt: Combine all delimited exprs as last argument
 
-This feature is not yet implemented.
-More information may be found in <https://github.com/rust-lang/rust/pull/114764>.
-
 ## Summary
 
+* Some expressions as the last argument in a list will now overflow instead of indent.
```

Comment:
perhaps: Some expressions with multi-line final arguments will now format as a single line, with the final expression overflowing

?? or something like that. I think the gist is that we're actually formatting more things as a single line + overflow.

Reply:
That looks better, I have updated with that suggestion.